lib/lib-schema-id/src/main/java/org/sagebionetworks/schema/id/OrganizationName.java

```diff
@@ -0,0 +1,11 @@
+package org.sagebionetworks.schema.id;
+
+import org.sagebionetworks.schema.element.SimpleBranch;
+
+public class OrganizationName extends SimpleBranch {
```

Comment:
Should extend DotSeparatedAlphaNumeric

Reply:
I tried to do this but it is not compatible with how the parser works.